extension/ts/workers/finder/impl/InternetArchive.ts

```diff
@@ -0,0 +1,7 @@
+import {Finder, FinderResponse} from "../finder";
+
+const InternetArchive: Finder = async (): Promise<FinderResponse> => {
+	return [];
+};
```

Comment:
is the plan to still do this later?

Reply:
yeah I hope so but not in the mood right now